package.json

```diff
@@ -98,12 +98,12 @@
     {
       "name": "Minimum",
       "import": "{ localeFrom, createI18n }",
-      "limit": "520 B"
+      "limit": "690 B"
     },
     {
       "name": "Maximum",
       "import": "{ localeFrom, browser, createI18n, params, count, formatter, createProcessor }",
-      "limit": "953 B"
+      "limit": "1.13 kB"
```

Comment:
Can you move back to byes (something like `1157 B`)? It will force Size Limit to show warning if limit it bigger than size.

Reply:
done